### ISSUE TITLE
Sunflower cataract

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10396]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
PMID: 35566651 Sunflower cataract is another classical ocular manifestation of WD, the frequency of which varies widely in the literature (between 2 and 20%) .... Sunflower cataract consists of copper deposition in the lens capsule and not within the lens cortex or nucleus itself, and has an aspect of a central disk with radiating petal-like spokes that give its name 

PMID: 34432061 The sunflower cataract, which presents as a centralized copper deposition in the anterior capsule surrounded by spoke-like radial cortical opacities, seldom impacts the patient’s visual acuity and occurred in about only 1.2% of untreated WD patients

**Parent term (use hpo.jax.org/app)**
Cataract HP:0000518

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Wilson disease ORPHA:905

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000642
- label: Sunflower cataract
- parents: [Anterior subcapsular cataract (HP:0010923)](https://hpo.jax.org/app/browse/term/HP:0010923)
- pmid: PMID:26577266|PMID:25609909
- def: Sunflower cataract (SC) is a type of anterior subcapsular cataract almost only seen in Wilson disease, an autosomal-recessive condition with a defect in the metabolism of copper leading to accumulation of copper in the liver and basal ganglia. SC consists of a thin, centralized opacification that is located directly under the anterior capsule and encompasses between one-third and one-half of the anterior lens pole surface area. In all cases, the central opacification is surrounded by additional, secondary opacifications arranged in ray-like structures around it. This pattern is said to resemble a sunflower, with a large central disk surrounded by petals.
- comment: 
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10396
- synonyms:
